docs/user-guide/rules.md

```diff
@@ -141,6 +141,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`selector-combinator-space-after`](../../src/rules/selector-combinator-space-after/README.md): Require a single space or disallow whitespace after the combinators of selectors.
 - [`selector-combinator-space-before`](../../src/rules/selector-combinator-space-before/README.md): Require a single space or disallow whitespace before the combinators of selectors.
 - [`selector-id-pattern`](../../src/rules/selector-id-pattern/README.md): Specify a pattern for id selectors.
+- [`selector-max-depth`](../../src/rules/selector-max-depth/README.md): Limit the depth of selectors.
```

Comment:
Sorry to just come up with this idea --- but might it be more clear to word all this as targeting *the number of combinators*: so the rule name could be `selector-max-combinators`, and the description: `Limit the number of combinators in selectors.`

If those words are equally or more accurate, I'd prefer to use them, because "depth" is vague and unspecified, but the quantity of combinators is precise.